sql/query/expr/like.go

```diff
@@ -0,0 +1,193 @@
+package expr
+
+import (
+	"errors"
+	"fmt"
+	"github.com/genjidb/genji/document"
+	"github.com/genjidb/genji/sql/scanner"
+	"regexp"
+	"strings"
+)
```

Comment:
We should keep conventional import grouping (`x/tools/cmd/goimports` or `mvdan.cc/gofumpt` may help with this).

```suggestion
import (
	"errors"
	"fmt"
	"regexp"
	"strings"
	
	"github.com/genjidb/genji/document"
	"github.com/genjidb/genji/sql/scanner"
)
```